planner/core/plan_cache.go

```diff
@@ -305,6 +305,26 @@ func RebuildPlan4CachedPlan(p Plan) error {
 	return rebuildRange(p)
 }
 
+func updateRange(p PhysicalPlan, ranges ranger.Ranges, rangeInfo string) error {
```

Comment:
Can this function return an error? If not, should we remove the returned error?

Reply:
Updated.